bangazonapi/views/profile.py

```diff
@@ -156,7 +156,7 @@ def cart(self, request):
 
                 cart = {}
                 cart["order"] = OrderSerializer(open_order, many=False, context={'request': request}).data
-                cart["order"]["line_items"] = line_items.data
+                # cart["order"]["line_items"] = line_items.data
```

Comment:
Don't leave in old commented code once you determine that it was the issue, make sure to completely remove it.

Reply:
Pushed new commit to address this. Line 159 deleted.